modules/account-lib/src/coin/hbar/ifaces.ts

```diff
@@ -6,3 +6,7 @@ export interface TxData {
   fee: number;
   startTime: string;
 }
+
+export interface HederaNode {
+  nodeId: string;
+}
```

Comment:
QUESTION: does this need to be account on node sensitive as well? there's also a realm attribute to the node-account-realm construct I think we might need to understand here

Reply:
Yes, we checked the SDK behaviour to setNodeAccountId and they check this in the same way, by using new AccountID(address). I documented that on the node() method in transaction builder saying that it may take the format `'<shard>.<realm>.<account>'` or `'<account>'`